README.md

```diff
@@ -215,3 +215,99 @@ The above listener will receive a `CursorUpdate` event:
   "data": { "color": "red" }
 }
 ```
+
+### Component Locking
+
+Use the Component Locking API to lock stateful components whilst being edited by members to reduce the chances of conflicting changes being made.
+
+Locks are identified using a unique string, and the Spaces SDK maintains that at most one member holds a lock with a given string at any given time.
+
+The Component Locking API supports four operations: Query, Acquire, Release, and Subscribe.
+
+### Query
+
+`space.locks.get` is used to query whether a lock identifier is currently locked and by whom. It returns a `Lock` type which has the following fields:
+
+```typescript
+type Lock = {
+  member: SpaceMember;
+  request: LockRequest;
+};
+```
+
+For example:
+
+```javascript
+// check if the id is locked
+const isLocked = space.locks.get(id) !== undefined;
+
+// check which member has the lock
+const { member } = space.locks.get(id);
+
+// check the lock attributes assigned by the member holding the lock
+const { request } = space.locks.get(id);
+const value = request.attributes.get(key);
+```
+
+`space.locks.getAll` returns all lock identifiers which are currently locked as an array of `Lock`:
+
+```javascript
+const allLocks = space.locks.getAll();
+
+for (const lock of allLocks) {
+  // ...
+}
+```
+
+### Acquire
+
+`space.locks.acquire` initialises a lock request, adds it to the `locks` field of presence message extras, and calls `presence.update`.
```

Comment:
I'm not sure the reader will understand "adds it to the locks field of presence message extras..", or needs to know.

Reply:
Yes true, reworded to "sends a request to acquire a lock using presence" in 644501d.